src/lib/utils.ts

```diff
@@ -30,3 +30,37 @@ export const managementAppRequest = async (): Promise<ManagementAppResponse> =>
     const data = await response.json()
     return data
 }
+
+// Functions for interacting with the Trusted Output App
+export const toaRequest = async (): Promise<TOAResponse> => {
```

Comment:
Maybe we could call this something like `getRuns` or something more descriptive (i'm bad with variable names 😭)

Reply:
I updated this and the corresponding type name to be more descriptive. I made a similar change for the corresponding BMA bits for consistency.